web/templates/layout/app.html.eex

```diff
@@ -16,6 +16,14 @@
   </head>
 
   <body class="w-100 sans-serif bg-white">
+
+    <header class="hl-bg-aqua w-100 ph3 pv3 pv4-ns ph4-m ph5-l tc">
+      <nav class="f4 fw3 tracked">
+        <i id="open-nav" class="fa fa-bars dim white dib fl" aria-hidden="true"></i>
+        <a href="/"><%= img_tag("/images/logo.svg", class: "w-70") %></a>
```

Comment:
I've got a horrible feeling we might need to support IE8, if so we'll need an SVG fallback.